src/node_dotenv.cc

```diff
@@ -52,6 +52,8 @@ void Dotenv::SetEnvironment(node::Environment* env) {
     auto existing = env->env_vars()->Get(key.data());
 
     if (existing.IsNothing()) {
+      // Remove all '\' characters from value
+      value.erase(std::remove(value.begin(), value.end(), '\\'), value.end());
```

Comment:
Should this be moved to `Dotenv::ParseLine` ? 🤔

Reply:
Thank you for your comment.
IMO, `Dotenv::ParseLine()` is used in `Dotenv::ParsePath()`, and the current issue does not seem to be about `Path`.

Reply:
Sorry, I checked further and found what you said was correct. I applied it.